specification/mediaservices/resource-manager/Microsoft.Media/Streaming/stable/2022-11-01/streamingservice.json

```diff
@@ -0,0 +1,2877 @@
+          "description": "The streaming endpoint default capacity."
+        },
+        "minimum": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The streaming endpoint minimum capacity."
+        },
+        "maximum": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The streaming endpoint maximum capacity."
+        }
+      },
+      "description": "The streaming endpoint sku capacity.",
+      "type": "object"
+    },
+    "ArmStreamingEndpointSkuInfo": {
+      "properties": {
+        "resourceType": {
+          "type": "string"
+        },
+        "capacity": {
+          "$ref": "#/definitions/ArmStreamingEndpointCapacity"
+        },
+        "sku": {
+          "$ref": "#/definitions/ArmStreamingEndpointSku"
+        }
+      },
+      "type": "object"
+    },
+    "StreamingEndpointSkuInfoListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ArmStreamingEndpointSkuInfo"
+          },
+          "x-ms-identifiers": [],
+          "description": "The result of the List StreamingEndpoint skus."
+        }
+      },
+      "type": "object"
+    },
+    "AsyncOperationResult": {
+      "properties": {
+        "error": {
+          "description": "The error object",
+          "$ref": "../../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorDetail"
+        },
+        "name": {
+          "type": "string",
+          "description": "Operation Id of the async operation."
+        },
+        "status": {
+          "description": "Operation status of the async operation.",
+          "type": "string",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "InProgress"
+          ],
+          "x-ms-enum": {
+            "name": "AsyncOperationStatus",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "The status of an async operation.",
+      "type": "object"
+    },
+    "IngestInterruption": {
+      "properties": {
+        "begin": {
+          "description": "UTC time of interruption start, encoder disconnected.",
+          "type": "string"
+        },
+        "end": {
+          "description": "UTC time of interruption end, encoder re-connected.",
+          "type": "string"
+        },
+        "durationInSec": {
+          "description": "Duration of interruption in seconds.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "reason": {
+          "description": "Interruption reason.",
+          "type": "string"
+        }
+      },
+      "description": "The live event ingest interruption data.",
+      "type": "object"
+    },
+    "Ingestion": {
+      "properties": {
+        "streamName": {
+          "description": "Ingestion stream name.",
+          "type": "string"
+        },
+        "beginTime": {
```

Comment:
The ingest interruption type uses begin/end while this type uses beginTime/endTime. Could these be made consistent?